src/vmshepherd/http/rpc_api.py

```diff
@@ -112,13 +113,25 @@ def wrap(self, *args, **kwargs):
         :rtype: string
         """
         vmshepherd = self.request.app.vmshepherd
-        preset = vmshepherd.preset_manager.get_preset(preset_name)
+        try:
+            preset = vmshepherd.preset_manager.get_preset(preset_name)
+        except Exception as ex:
+            logging.error(ex)
```

Comment:
I don't get it. 
I believe the stack trace is being pushed to `stderr` when an exception is raised.
What is the purpose of logging the same thing twice?

Reply:
to get rid of stack trace, to be able to read log line in let's say clear form

Reply:
Well I think, that make sense only in case of exception shadowing. When you want to raise custom exception but keep log of a original exception.
In this case we multiply the error message, which make no sense for me.

Reply:
We've figured out, that in this case, the jsonrpc handling library is suppressing all the errors. In such a case, we've decided to go with the decorator that will catch all the errors, log and reraise them. Please see 4750f9c